source/includes/_orders.md

```diff
@@ -0,0 +1,773 @@
+# Orders
+
+## Place Purchase Order
+
+> Place Purchase Order Request
+
+```json
+{
+  "userId": "test_user_id",
+  "groupId": "test_group_id",
+  "productCode": "pid_1234567890",
+  "orderType": "purchase",
+  "paymentAmount": 100,
+  "schedule": {
+    "startTimestamp": 1735718400,
+    "endTimestamp": 1738396800
+  },
+  "contentItemIds": [],
+  "variableValues": {
+    "accountId": "1234567890",
+    "pageId": "1234567890",
+    "headline": "headline test",
+    "bodyText": "body text test",
+    "image": "https://image.test",
+    ...
+  }
+}
+```
+
+> Place Purchase Order Response
+
+```json
+{
+  "data": {
+    "id": "1234567890",
+    "userId": "test_user_id",
+    "groupId": "test_group_id",
+    "productCode": "pid_1234567890",
+    "status": "pending",
```

Comment:
ah I used UPPERCASE enums for this status. i can turn it into lowercase if needed, all the other enum fields like orderType and subscription status are using lowercase too may help to be consistent

Reply:
sounds good to me.

*also, per your comment on the other PR, we're keeping it lowercase.

Reply:
I'll also change it so all order statuses match correctly with your updates. Currently, I have it set to whatever `order_item.status` is